tests/wdio/application-base/application-base-spec.js

```diff
@@ -1,51 +1,39 @@
 Terra.describeViewports('ApplicationBase', ['small', 'large'], () => {
-  describe('breakpoint', () => {
-    before(() => {
-      browser.url('/raw/tests/terra-application/application-base/application-base-test');
-      browser.refresh();
-    });
+  it('should render the active breakpoint', () => {
+    browser.url('/raw/tests/terra-application/application-base/application-base-test');
 
-    Terra.it.validatesElement('with breakpoint', { selector: '#root' });
+    Terra.validates.element('active breakpoint', { selector: '#root' });
   });
 
-  describe('intl', () => {
-    before(() => {
-      browser.url('/raw/tests/terra-application/application-base/application-base-test');
-      browser.refresh();
-    });
+  it('should render the active locale', () => {
+    browser.url('/raw/tests/terra-application/application-base/application-base-test');
 
-    Terra.it.validatesElement('with intl', { selector: '#root' });
+    Terra.validates.element('active locale', { selector: '#root' });
```

Comment:
I think you could combine these two tests since the only difference is screenshot name.

Reply:
Combined: https://github.com/cerner/terra-application/pull/117/commits/cd921349a7fedd3396fc9bbb1328521a2653be0b